egui_demo_lib/src/apps/demo/drag_and_drop.rs

```diff
@@ -116,18 +119,23 @@ impl super::View for DragAndDropDemo {
 
         let mut source_col_row = None;
         let mut drop_col = None;
-
         ui.columns(self.columns.len(), |uis| {
-            for (col_idx, column) in self.columns.iter().enumerate() {
+            for (col_idx, column) in self.columns.clone().into_iter().enumerate() {
                 let ui = &mut uis[col_idx];
                 let can_accept_what_is_being_dragged = true; // We accept anything being dragged (for now) ¯\_(ツ)_/¯
                 let response = drop_target(ui, can_accept_what_is_being_dragged, |ui| {
                     ui.set_min_size(vec2(64.0, 100.0));
-
-                    for (row_idx, &item) in column.iter().enumerate() {
+                    for (row_idx, item) in column.iter().enumerate() {
                         let item_id = Id::new("item").with(col_idx).with(row_idx);
                         drag_source(ui, item_id, |ui| {
-                            ui.label(item);
+                            ui.add(Label::new(item).sense(Sense::click())).context_menu(
+                                |ui, menu_state| {
+                                    if menu_state.item("Remove...").show(ui).clicked() {
```

Comment:
```suggestion
                                    if menu_state.item("Remove").show(ui).clicked() {
```

An ellipsis (…) is used to signify that additional information will be collected, which is not the case here.